pkgs/build-support/fetchurl/default.nix

```diff
@@ -132,6 +132,8 @@ else stdenv.mkDerivation {
 
   impureEnvVars = impureEnvVars ++ netrcImpureEnvVars;
 
+  nixpkgsVersion = fileContents ../../../.version;
```

Comment:
Why not just export the major version from `lib`?

Reply:
`lib` doesn't export major version specifically, only suffixed version: https://github.com/NixOS/nixpkgs/blob/b5a61f2c599ac665e3c3129258ea8671cb5760af/lib/trivial.nix#L62-L65